babel.config.js

```diff
@@ -24,7 +24,7 @@ module.exports = (api) => {
             "@babel/preset-env",
             {
               targets: {
-                node: "14.15.0",
+                node: "16.0.0",
```

Comment:
I think we should use `16.10.0` (stable LTS)

Reply:
Done.